presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -0,0 +1,174 @@
+=======================
+Elasticsearch Connector
+=======================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+The Elasticsearch Connector allows access to Elasticsearch data from Presto. This document describes how to setup the Elasticsearch Connector to run SQL on Elasticsearch.
+
+.. note::
+
+    It is highly recommended to use Elasticsearch 6.0.0 or later.
+
+Configuration
+-------------
+
+To configure the Elasticsearch connector, create a catalog properties file
+``etc/catalog/elasticsearch.properties`` with the following contents,
+replacing the properties as appropriate:
+
+.. code-block:: none
+
+    connector.name=elasticsearch
+    elasticsearch.table-names=schema.table1,schema.table2
+    elasticsearch.default-schema=default
+    elasticsearch.table-description-dir=etc/elasticsearch/
+    elasticsearch.scroll-size=1000
+    elasticsearch.scroll-time=60000
+
+Configuration Properties
+------------------------
+
+The following configuration properties are available:
+
+======================================= ==============================================================================
+Property Name                           Description
+======================================= ==============================================================================
+``elasticsearch.table-names``           List of all tables provided by the catalog
+``elasticsearch.default-schema``        Default schema name for tables
+``elasticsearch.table-description-dir`` Directory containing JSON table description files
+``elasticsearch.scroll-size``           Maximum number of hits to be returned with each batch of Elasticsearch scroll
+``elasticsearch.scroll-timeout``        Number of milliseconds Elasticsearch would keep live for a search scroll result
```

Comment:
It's keeping alive search contexts in the backend, so we should mention that.

```
Amount of time (ms) Elasticsearch will keep the search context alive for scroll requests
```

It may also be useful to link to the relevant sections of the ES docs, like the one you mention, so that admins/users can understand what he/she is really setting.